src/Backend/Modules/Settings/Actions/Index.php

```diff
@@ -643,10 +643,14 @@ private function validateForm(): void
                     'show_consent_dialog',
                     $this->form->getField('show_consent_dialog')->getChecked()
                 );
+                $privacyConsentLevels = [];
+                if($privacyConsentLevelsField->isFilled()) {
+                    $privacyConsentLevels = explode(',', $this->form->getField('privacy_consent_levels')->getValue());
```

Comment:
You can use $privacyConsentLevelsField here

Reply:
Fixed in b31a09ce909c0486bffd97eacf51274cf23539fd